### DISCUSSION TITLE
Monitor OpenWiFi nodes

### DISCUSSION THREAD

**Discussion by jaller94** [windmill-labs/windmill#583] [Show and tell]
Happy Friday, everyone!
In the Matrix community there's a habit of showing off something one has done during the week.
I think that could be fun for Windmill as well.

This week, I've wanted to monitor the mesh quality of an important Public WiFi router in my area.
Freifunk is a community-built WiFi network in Germany. Router can have their own Internet uplink and/or mesh with other routers.
To monitor the connection quality over time, I've built a Schedule that fetches the mesh links from an HTTP service and posts them into a Matrix room.
In the future I want to also add a Trigger Script that alerts me when a mesh connection is lost or its quality decreased a lot.

![Screenshot_20220916-170015](https://user-images.githubusercontent.com/10872136/190692960-7aa197fc-ae85-4152-9a15-1e874b71041e.jpg)

**Comment by rubenfiszel** [comment]
Love  it.

When you feel like it's ready to share, feel free to add your flow to windmillhub: https://hub.windmill.dev/flows/add !
